src/shogun/base/SGObject.h

```diff
@@ -441,15 +441,15 @@ class CSGObject
 	/** Typed array getter for an object array class parameter of a Shogun base class
 	* type, identified by a name and an index.
 	*
-	* Raises an error if parameter does not exist.
+	* It does not throw any errors if it does not work.
```

Comment:
minor: awkward english due to double negation. "Returns nullptr if parameter of desired type does not exist"